LiskHQ/lisk-hub
# Implement minimal BTC wallet page - Closes #1888

### What issue have I solved?

#1888

### How have I implemented/fixed it?

- I updated `walletTransactionsV2` HOC to get data based on `activeToken`.
- I updated various components not to show if token is BTC

There are two refactoring changes not essential to the problem at hand
- 🔥 Remove functionMapper to make code cleaner https://github.com/LiskHQ/lisk-hub/pull/2003/commits/d2f8280c1c31995d60bd8543935533ff272a23d2
- ♻️ Move updateTransactionsIfNeeded to transactions actions https://github.com/LiskHQ/lisk-hub/pull/2003/commits/02b2ae4fe29f48bc802ded7bceb5cd436ee2fe24
### How has this been tested?

- Go to wallet
- Open React dev tools (because of https://stackoverflow.com/a/44318447/2080716)
- Enable BTC feature `localStorage.setItem('btc', true);`
- Set BTC active token by running this in console `$r.store.dispatch({type:'SETTINGS_UPDATED', data: { token: { active: 'BTC', list: {BTC: true, LSK: true} }}}); ` Note that React dev tools have to be active for `$r` to be defined.
- Switch active token back to LSK  by running this in console `$r.store.dispatch({type:'SETTINGS_UPDATED', data: { token: { active: 'LSK', list: {BTC: true, LSK: true} }}}); `. 

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
